nautobot_firewall_models/models/through_models.py

```diff
@@ -149,3 +149,171 @@ class ZoneVRFM2M(BaseModel):
 
     zone = models.ForeignKey("nautobot_firewall_models.Zone", on_delete=models.CASCADE)
     vrf = models.ForeignKey("ipam.vrf", on_delete=models.PROTECT)
+
+
+class NATPolicyNATRuleM2M(BaseModel):
```

Comment:
I am wondering IF we need this for NAT as well?

Reply:
I would argue that since the NAT models follow the security models rather closely, any deviation in behavior could lead to unexpected behavior from the user side, what do you think?